Snakefile

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 import sys
+import yaml
```

Comment:
Can we drop this line? it seems is not needed

Reply:
True, have forgotten to remove it. Thanks!